gradle_plugin/src/main/groovy/com/microsoft/hydralab/entity/AttachmentInfo.java

```diff
@@ -0,0 +1,9 @@
+package com.microsoft.hydralab.entity;
```

Comment:
Please attach this comment to each new file.

// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.